interface.go

```diff
@@ -234,6 +234,48 @@ type LoopInRequest struct {
 	RouteHints [][]zpay32.HopHint
 }
 
+// StaticAddressLoopInRequest contains the required parameters for the swap.
+type StaticAddressLoopInRequest struct {
+	// DepositOutpoints contain the outpoints of the static address deposits
+	// that are being looped in. The sum of output values constitute the
+	// swap amount.
+	DepositOutpoints []string
+
+	// MaxSwapFee is the maximum we are willing to pay the server for the
+	// swap. This value is not disclosed in the swap initiation call, but if
+	// the server asks for a higher fee, we abort the swap. Typically, this
+	// value is taken from the response of the LoopInQuote call. It
+	// includes the pre-pay amount.
+	MaxSwapFee btcutil.Amount
+
+	// LastHop optionally specifies the last hop to use for the loop in
+	// payment.
+	LastHop *route.Vertex
+
+	// Label contains an optional label for the swap.
```

Comment:
Describe the set of allowed characters.

Reply:
Not quite sure, this should be UTF-8 char set. I added that this is a `text label`.